README.md

```diff
@@ -1,166 +1,166 @@
+
+- `latest` -- the most recent version corresponding to [the master branch](https://github.com/ManimCommunity/manim)
+- `stable` -- the latest released version (according to [the releases page](https://github.com/ManimCommunity/manim/releases))
+- `vX.Y.Z` -- any particular released version (according to [the releases page](https://github.com/ManimCommunity/manim/releases))
+
+### Instructions for running the docker image
+
+#### Quick Example
+To render a scene `CircleToSquare` in a file `test_scenes.py` contained in your current working directory while preserving your user and group ID, use
+```
+docker run --rm -it  --user="$(id -u):$(id -g)" -v "$(pwd)":/manim manimcommunity/manim manim test_scenes.py CircleToSquare -qm
+```
+
+#### Running the image in the background
+Instead of using the "throwaway container" approach sketched above, you can also create a named container that you can also modify to your liking. First, run
+```
+docker run -it --name my-manim-container -v "$(pwd):/manim" manimcommunity/manim /bin/bash
+```
+to obtain an interactive shell inside your container allowing you to, e.g., install further dependencies (like texlive packages using `tlmgr`). Exit the container as soon as you are satisfied. Then, before using it, start the container by running
+```
+docker start my-manim-container
+```
+Then, to render a scene `CircleToSquare` in a file `test_scenes.py`, call
+```
+docker exec -it --user="$(id -u):$(id -g)" my-manim-container manim test.py CircleToSquare -qm
+```
+
+#### Jupyterlab
+Another alternative is to use the docker image to spin up a local webserver running
+JupyterLab in whose Python kernel manim is installed and can be accessed via the `%%manim` cell magic.
+To use JupyterLab, run
+```
+docker run -it -p 8888:8888 manimcommunity/manim jupyter lab --ip=0.0.0.0
+```
+and then follow the instructions in the terminal.
+
+#### Important notes
+
+When executing `manim` within a Docker container, several command line flags (in particular `-p` (preview file) and `-f` (show output file in the file browser)) are not supported.
+
+## Help with Manim
+
+If you need help installing or using Manim, feel free to reach out to our [Discord
+Server](https://discord.gg/mMRrZQW) or [Reddit Community](https://www.reddit.com/r/manim). If you would like to submit a bug report or feature request, please open an issue.
+
+## Contributing
+
+Contributions to Manim are always welcome. In particular, there is a dire need for tests and documentation. For contribution guidelines, please see the [documentation](https://docs.manim.community/en/stable/contributing.html).
+
+Most developers on the project use [Poetry](https://python-poetry.org/docs/) for management. You'll want to have poetry installed and available in your environment. You can learn more about `poetry` and how to use it at its [documentation](https://docs.manim.community/en/stable/installation/for_dev.html).

```

Comment:
All the changes to the README are in between those lines, I guess the problem was that my machine accidentally replaced all the line-endings